src/test/scala/io/qbeast/spark/delta/keeper/RandomKeeper.scala

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.qbeast.spark.keeper
+package io.qbeast.spark.delta.keeper
```

Comment:
I don't think the keeper should be in the delta package...

Reply:
The keep logic remains in the `core/` folder; however, I moved the keeper tests to the delta tests folder because they currently strongly depend on delta operations, and I didn't see an easy way to modify the tests to avoid using those delta operations